viscm/gui.py

```diff
@@ -871,27 +956,31 @@ def loadpyfile(path):
     is_native = True
     cmtype = "linear"
     method = "Bezier"
-    ns = {'__name__': '',
-          '__file__': os.path.basename(path),
-          }
+    ns = {
+        "__name__": "",
+        "__file__": os.path.basename(path),
+    }
+
+    # flake8: noqa
+    # FIXME: Should be `args.colormap` should be `path`?
     with open(args.colormap) as f:
-        code = compile(f.read(),
-                       os.path.basename(args.colormap),
-                       'exec')
+        # flake8: noqa
+        code = compile(f.read(), os.path.basename(args.colormap), "exec")
```

Comment:
`flake8` caught what looks like a bug here. `args` is not available to this function. Seems clear to me at the moment it should be path, but I'm not sure the best way to test yet. I'd like to eventually fix this bug and add a unit test for the python file loading mechanism, but I think that's best suited for another PR.